recipes/libbacktrace/all/conanfile.py

```diff
@@ -49,6 +59,9 @@ def build(self):
     def package(self):
         self._autotools.install()
         self.copy("LICENSE", dst="licenses")
+        la = os.path.join(self.package_folder, "lib", "libbacktrace.la")
+        if os.path.exists(la):
+            os.unlink(la)
```

Comment:
You can use this newer feature

https://github.com/conan-io/conan-center-index/commit/1b5321fa92d209203721febbb70504b97cc38dc6#diff-
7a002ca818a070c9265f9504710a98d337d6fc88fa1d907cb01b709f0b3975caR4-R66

Reply:
Thanks!